auth_ldap_attribute_sync/views/res_company_ldap_views.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!--
+    Copyright (C) 2018 Brainbean Apps (https://brainbeanapps.com)
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+    -->
+    <record id="view_ldap_installer_form" model="ir.ui.view">
+        <field name="name">res.company.ldap.form</field>
+        <field name="model">res.company.ldap</field>
+        <field name="inherit_id" ref="auth_ldap.view_ldap_installer_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group/field[@name='ldap_server']/../..">
+                <group string="Map User Attributes" >
+                    <field name="user_attributes_mapping" nolabel="1"/>
+                </group>
+            </xpath>
```

Comment:
Remark/question: there is no `position` keyword, does that mean the default is `after`? Or this code is buggy?

Reply:
Default `position` is `inside`, so it seems correct.

Reply:
@pedrobaeza oh I didn't know, thank you!

Reply:
Right, `inside` is the default:
"the content of the inheritance spec is appended to the matched node" from [here](https://www.odoo.com/documentation/12.0/reference/views.html#inheritance-specs)